minecraft_servers/brawl/manifest.json

```diff
@@ -0,0 +1,55 @@
+{
+  "server_name": "brawl",
+  "nice_name": "Brawl Games",
+  "direct_ip": "brawl.com",
+  "server_wildcards": [
+    "%.brawl.com"
+  ],
+  "social": {
+    "web": "https://www.brawl.com/",
+    "web_shop": "https://shop.brawl.com/",
+    "web_support": "https://www.brawl.com/support/",
+    "twitter": "BrawlGames",
+    "facebook": "BrawlGames",
+	"tiktok":  "brawlwastaken",
+    "instagram": "minecraft_brawl",
+    "youtube": "https://www.youtube.com/channel/UCDHRIcT3eQ9HEFlqGtv_9kQ",
+    "discord": "https://discord.brawl.com"
```

Comment:
Please have a look at [Manifest.md#discord-url](https://github.com/LabyMod/server-media/blob/master/docs/Manifest.md#discord-url):
Please change this URL to an official discord invite link (e.g. https://discord.gg/xyz )